telemetry/definitions/commonDefinitions.json

```diff
@@ -982,6 +982,21 @@
             "type": "string",
             "description": "The API request ID"
         },
+        {
+            "name": "codeTransformMavenCommands",
+            "type": "string",
+            "description": "Maven commands that have been run."
+        },
+        {
+            "name": "codeTransformMavenExecutionSuccessful",
```

Comment:
They won't be able to even upload the project / start the job in either IDE if any Maven command fails

Reply:
That make sense. I removed the field.